Libraries/LibWeb/CSS/StyleValues/CSSColorValue.cpp

```diff
@@ -33,7 +33,10 @@ Optional<double> CSSColorValue::resolve_hue(CSSStyleValue const& style_value)
 {
     // <number> | <angle> | none
     auto normalized = [](double number) {
-        return fmod(number, 360.0);
+        auto result = fmod(number, 360.0);
+        if (number < 0)
```

Comment:
This is same as `JS::modulo`, perhaps it makes sense to put something like that in AK (or just use directly since this file will have that pulled in somewhere anyhow)

Reply:
My latest push uses this function directly :^)